Analyse_Birds.qmd

```diff
@@ -107,13 +117,220 @@ required_packages <- c(
   "lattice",    # for multipanel graphics
   "dplyr",      # for data manipulation
   "FactoMineR", # for ordinations
-  "factoextra"
+  "factoextra", 
+  "sf",         # for coordinats manipulation
+  "terra",      # for maps creation
+  "spData", 
+  "tmap", 
+  "leaflet", 
+  "spDataLarge", 
+  "maps", 
+  "gifski"      # for gifs display
 )
 
 # Install and load the packages
 install_if_missing(required_packages)
 ```
 
+# First look at the data : 
+
+You can visualise the first rows of the dataset by running the following cell :
+```{r}
+# View the firsts lignes of the dataset
+head(data)
+```
+
+In order to picture the data in a more visual way, we will be representing our dataset on a map. 
+The first step in our map creation is the separation of the dataset by the counties. 
+
+```{r}
+data_list_county <- split(data, data$County)
+
+data_will <- data_list_county[["Will"]]
+data_jo_daviess <- data_list_county[["Jo Daviess"]]
+data_lafayette <- data_list_county[["Lafayette"]]
+data_iowa <- data_list_county[["Iowa"]]
+data_dane <- data_list_county[["Dane"]]
+data_green <- data_list_county[["Green"]]
+data_grant <- data_list_county[["Grant"]]
+data_grundy <- data_list_county[["Grundy"]]
+data_lee_ogle <- data_list_county[["Lee/Ogle"]]
+data_lee <- data_list_county[["Lee"]]
+data_rock <- data_list_county[["Rock"]]
+data_monroe <- data_list_county[["Monroe"]]
+data_carroll <- data_list_county[["Carroll"]]
+data_carroll_whiteside <- data_list_county[["Carroll/Whiteside"]]
+```
+
+We are now adding columns for the coordinates of each county, the coordinates were obtained by searching for the conties on openstreetmap. 
```

Comment:
"counties" not conties

Reply:
This typo has been updated, thank you for spotting it